python.sh

```diff
@@ -54,7 +54,7 @@ esac
 if [[ $OPENSSL_ROOT ]]; then
   export CPATH="$OPENSSL_ROOT/include:$OPENSSL_ROOT/include/openssl:$CPATH"
   export CPPFLAGS="-I$OPENSSL_ROOT/include -I$OPENSSL_ROOT/include/openssl $CPPFLAGS"
-  export CFLAGS="-I$OPENSSL_ROOT/include -I$OPENSSL_ROOT/include/openssl"
```

Comment:
Maybe what we should do is really drop CFLAFS, since most likely CPPFLAGS is enough. Could you check?

Reply:
Ok, then it's needed...